docs/integration/solr.md

```diff
@@ -1,8 +1,114 @@
 title: Solr Monitoring Integration
 description:  Our monitoring and logging platform includes integration for Solr. Use predefined key metrics reports combined with rich data visualization tools to monitor critical Solr issues, and receive alerts on memory usage, uptime, load averages, index stats, document and filter caches, latency, rate, and more
 
+Sematext offers simple and versatile Solr monitoring agent written in Java and Golang without CPU and memory overhead on your application. It's easy to install and require no changes in Solr source code or your application source code. 
+
+## Sematext Solr Monitoring Agent 
+This lightweight, open-source [Java Monitoring Agent](https://github.com/sematext/sematext-agent-java) collects Kafka performance metrics and sends them to Sematext. It comes packaged with a Golang-based agent responsible for Operating System level metrics like network, I/O and more. The Solr Monitoring Agent can be installed with RPM/DEB package manager on any host running Linux or in a containerized environment using ```sematext/spm-client```.
+
+The Sematext SolrCloud Monitoring Agent can be run in two different modes - *in-process* and *standalone*. The *in-process* one is run as a Java agent, it is simpler to initially set up, but will require restarting your Solr node when you will want to upgrade your monitoring Agent, i.e. to get new features. The benefit of the *standalone* agent mode is that it's running as a separate process and doesn't require a Solr restart when it is upgraded.
+
+After creating a [Solr App in Sematext](https://apps.sematext.com/ui/monitoring-create) you need to install the Monitoring Agent on each host that is running Solr that you want to monitor. The full installation instructions can be found in the [setup instructions](https://apps.sematext.com/ui/howto/Solr/overview) displayed in the UI. 
+
+For example, on Ubuntu, you need to add Sematext Linux packages and install them with the following command:
+```bash
+echo "deb http://pub-repo.sematext.com/ubuntu sematext main" | sudo tee
+/etc/apt/sources.list.d/sematext.list > /dev/null
+wget -O - https://pub-repo.sematext.com/ubuntu/sematext.gpg.key | sudo apt-key add -
+sudo apt-get update
+sudo apt-get install spm-client
+``` 
+
+After that you need to setup the Solr Monitoring Agent by running a command like this:
```

Comment:
After that you need to setup => After that, setup

Reply:
Done.